examples/radial_gradient_filters.py

```diff
@@ -0,0 +1,57 @@
+"""
+=======================
+Radial Gradient Filters
+=======================
+
+This example applies both the normalizing radial gradient (`sunkit_image.radial.nrgf`) filter and Fourier
+normalizing radial gradient filter (`sunkit_image.radial.fnrgf`) to a sunpy map.
+
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.map
+import sunpy.data.sample
+
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+###########################################################################
+
+# Sunpy's sample data contain a number of suitable FITS files for this purpose.
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###########################################################################
+
+# Radial bins are calculated over which the filters will be applied. The distance
+# between 1 Solar radius and 2 Solar radius is divided into 100 equal parts.
+radial_bin_edges = equally_spaced_bins()
+radial_bin_edges *= u.R_sun
+
+# The NRGF filter is applied after it.
+out1 = radial.nrgf(aia_map, radial_bin_edges)
+
+# The NRGF filtered map is plotted. 
+out1.plot()
```

Comment:
As you said, the first image is very washed out, so you will want to change some of the plotting settings to make it look nicer and leave a comment about it.

Reply:
Also we will want to use the second image instead. You can do that by adding: `# sphinx_gallery_thumbnail_number = 2` into the example file.

Reply:
Okay